ui-core/src/main/res/template/values/strings.xml.tt

```diff
@@ -53,6 +53,7 @@
     <string name="checkout_address_lookup_empty">%%address.lookup.search.empty.title.noResults%%</string>
     <string name="checkout_address_lookup_empty_description">%%address.lookup.search.empty.subtitle.noResults%%</string>
     <string name="checkout_address_lookup_enter_manually">%%address.enterManually%%</string>
+    <string name="checkout_address_lookup_submit">%%address.lookup.submit%%</string>
```

Comment:
There's a comment in ui-core styles.xml that says: `<!-- TODO address lookup translations -->`
Was that about these translations?

Reply:
that was probably the case. looks like i missed it before merging the PR.